modules/ossm-configuring-the-threescale-wasm-auth-module.adoc

```diff
@@ -64,5 +63,5 @@ $ oc apply -f threescale-wasm-auth-bookinfo.yaml
 
 [role="_additional-resources"]
 .Additional resources
-* xref:../../service_mesh/v2x/ossm-extensions.adoc#ossm-smextensions-deploy_ossm-extensions[Deploying `ServiceMeshExtension` resources]
+* xref:../../service_mesh/v2x/ossm-extensions.adoc#ossm-extensions-migration-overview_ossm-extensions[Migrating from `ServiceMeshExtension` to `WasmPlugin` resources]
```

Comment:
The additional resources section should be in the assembly file  where this module is located. In this case it looks like it should be here, `service_mesh/v2x/ossm-threescale-webassembly-module.adoc`. Also our toolset does not allow `xrefs` in modules, which is why the additional resources section is in the Assembly file.

Reply:
@kelbrown20 Thanks for the review. I'm updating now.